### DISCUSSION TITLE
Error while doing reprojection

### DISCUSSION THREAD

**Discussion by Abresh27** [giswqs/geog-312#104] [Q&A] [answered]
I encountered the following error while I was doing reprojection on the raster data given in the lab_06
![Screenshot (6)](https://github.com/user-attachments/assets/3e8a03b0-1dde-447d-a6f8-afac400eb17d)

**Comment by giswqs** [comment, marked as answer]
This may be a GDAL installation issue on your computer. Try Google Colab.
